Testing/ContinuousIntegration/AzurePipelinesLinux.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           BUILD_EXAMPLES:BOOL=OFF
           ITK_WRAP_PYTHON:BOOL=OFF
         ")
+        set(CTEST_TEST_ARGS EXCLUDE_LABEL BigIO)
```

Comment:
I think this should go into the LinuxLegacyRemoved and LinuxCxx20 jobs and not the Linux one based on recent CI failures.